kanidm_unix_int/src/cache.rs

```diff
@@ -731,13 +732,15 @@ impl CacheLayer {
             .await
         {
             Ok(Some(n_tok)) => {
-                debug!("online password check success.");
+                event!(Level::DEBUG, "online password check success.");
```

Comment:
Nope, no benefit at all. I was thinking of changing macro calls to FQS like `tracing::debug!(...)`, but I'm not sure if that's a good idea because the `env_logger`'s macros seem to work fine for `tracing`.

Reply:
Sounds like it's best to leave it with the original things in that case :)